src/service/map.service.js

```diff
@@ -0,0 +1,19 @@
+import { encodeQueryString } from "../helpers/utils";
+
+export const MapService = {
```

Comment:
Instead of creating service, follow the "feature folder struture from redux toolkit and keep all your thunks in one file "

Reply:
changed to the above file structure model